centralledger/chart-handler-admin-transfer/values.yaml

```diff
@@ -0,0 +1,704 @@
+# Default values for central-ledger.
+# This is a YAML-formatted file.
+
+# Declare global configurations
+global:
+  config:
+    ## Pod scheduling preferences.
+    ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+    ##
+    affinity: {}
+
+    ## Node labels for pod assignment
+    ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+    nodeSelector: {}
+
+    ## Set toleration for schedular
+    ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+    tolerations: []
+
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+containers:
+  api:
+    image:
+      repository: mojaloop/central-ledger
+      tag: v3.8.1-snapshot
+      pullPolicy: Always
+      command: '["node", "src/handlers/index.js", "handler", "--prepare"]'
```

Comment:
This should be updated. Should this not be --admin?

Reply:
updated